ci/init-db.sh

```diff
@@ -1,27 +1,30 @@
 #!/usr/bin/env bash
-# initialize jupyterhub databases for testing
+# This script initialize multiple databases for testing in a running database
+# server, using either the mysql client or psql client with the root or postgres
+# users.
 
 set -eu
 
-MYSQL="mysql --user root --host $MYSQL_HOST --port $MYSQL_TCP_PORT -e "
-PSQL="psql --user postgres -c "
+# These environment variables influence the mysql and psql clients
+export MYSQL_HOST=${MYSQL_HOST:-127.0.0.1}
+export MYSQL_TCP_PORT=${MYSQL_TCP_PORT:-13306}
+export PGHOST=${PGHOST:-127.0.0.1}
+export PGPORT=${PGPORT:-5432}
```

Comment:
These lines don't do anything, do they? Those arguments are not used in this script anymore, they are used by the clients. This effectively overrides the default implementation in the clients themselves, hardcoding new defaults (that are presumably the same), but I think these `exports` can all be removed.